orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/OrchSessionService.java

```diff
@@ -158,6 +158,10 @@ public Optional<OrchSessionItem> getSessionFromRequestHeaders(Map<String, String
         return sanitiseBase64(sessionId).flatMap(id -> getSession(sessionId));
     }
 
+    public void deleteStoredOrchSession(String sessionId) {
```

Comment:
We seem to just refer to the Orch session as "session" in the context of OrchSessionService. So maybe just `deleteSession()`?